app/components/CardBet.tsx

```diff
@@ -83,42 +83,71 @@ export const CardBet = ({ userBets, children }: CardBetProps) => {
           <div className="flex items-center justify-between">
-            <div className="flex space-x-2">
+            <Tag colorScheme="quaternary" size="sm" className="capitalize">
+              {market.fpmm.category}
+            </Tag>
+            <div className="flex items-center space-x-2">
               <Tag colorScheme="quaternary" size="sm" className="capitalize">
-                {market.fpmm.category}
-              </Tag>
-              <Tag colorScheme="success" size="sm">
-                {position.getOutcome()}
+                {market.getMarketStatus()}
               </Tag>
+              <p className="text-sm text-text-low-em">
+                {remainingTime(market.closingDate)}
+              </p>
             </div>
-            <p className="text-sm text-text-low-em">
-              {remainingTime(market.closingDate)}
-            </p>
           </div>
-          <div className="flex space-x-4">
+          <div className="flex space-x-4 py-2">
             <MarketThumbnail
               width={40}
               height={40}
               className="size-[40px] rounded-8"
               marketId={market.fpmm.id}
             />
-            <div className="text-normal h-[80px] flex-1 overflow-y-auto font-semibold text-text-high-em md:text-xl">
-              {market.fpmm.title}
+            <div className="space-y-2">
+              <div className="flex-1 overflow-y-auto text-md font-semibold text-text-high-em md:text-xl">
+                {market.fpmm.title}
+              </div>
+              {market.isClosed && (
+                <div className="flex space-x-2">
+                  <p className="text-text-med-em">Answer:</p>
+                  {market.isAnswerInvalid || market.isCurrentAnswerInvalid ? (
+                    <Tag colorScheme="quaternary">No valid answer</Tag>
+                  ) : (
+                    market.currentAnswer !== null && (
+                      <Tag
+                        colorScheme={
+                          market.getCurrentAnswerOutcome()?.index === 0
+                            ? 'success'
+                            : 'danger'
+                        }
+                      >
+                        {market.getCurrentAnswerOutcome()?.name}
+                      </Tag>
+                    )
+                  )}
+                </div>
+              )}
             </div>
           </div>
         </section>
       </Link>
-      <section className="flex items-center border-t border-outline-base-em px-4 py-2 md:h-[48px] md:py-0">
+      <section className="flex items-center border-t border-outline-base-em p-4 md:h-[48px]">
         <div className="flex w-full items-center justify-between space-x-4">
-          <div className="flex flex-col items-start space-y-0.5 md:flex-row md:items-center md:space-x-3">
+          <div className="flex flex-col items-start space-y-1.5 md:flex-row md:items-center md:space-x-3 md:space-y-0">
```

Comment:
we probably can use gap here, so we don't need to md:space-y-0. 
Also I think we should increase the space x on md:, hence the `md:gap-5`

```suggestion
          <div className="flex flex-col items-start gap-y-1.5 md:flex-row md:items-center md:gap-5">
```

Reply:
Used close values to ones you suggested